content/reference/sql/access-management.md

```diff
@@ -0,0 +1,289 @@
+Access management in Dolt is handled similarly to how it is handled in MySQL.
+When Dolt is running in server mode (via [`sql-server`](../cli.md#dolt-sql-server)), it relies upon the grant tables (`mysql.user`, `mysql.db`, etc.) to control user access.
+Access is determined by the privileges that a user has.
+For more information on the basics of how users and privileges work and how to use them, [please read our blog post from when we announced their inclusion into Dolt](https://www.dolthub.com/blog/2022-02-16-introducing-users-and-privileges/).
+This document will assume some familiarity with users and privileges.
+
+## Enabling Users
+
+### Server Mode Required
+
+Privilege access checks are only enabled when running in [`sql-server`](../cli.md#dolt-sql-server) mode.
+All other commands, including all permutations of [`sql`](../cli.md#dolt-sql), operate with privilege access checks disabled.
+Currently, the only way to remotely interact with a dolt database is when a database is running under server mode, therefore it is the only context where it is sensible for users to exist.
+As all other commands require file-level access to a database's files, any possible protection offered by other commands enabling users could easily be circumvented by directly modifying the data within the database files.
+
+In addition to privilege checks only being enabled when running in server mode, any changes to the grant tables will only persist in server mode.
+If any commands that modify the grant tables are used outside the server mode, those commands will fail, as the grant tables will not exist (they're only created when in server mode).
+
+### CLI Argument
+
+To enable full user support when using [`sql-server`](../cli.md#dolt-sql-server) mode, you may pass in the `--privilege-file="PATH"` argument.
+`"PATH"` represents the path to the privileges file, generally named `privs.json`.
+
+### YAML Configuration Option
+
+To enable full user support when using [`sql-server`](../cli.md#dolt-sql-server) mode, you may add the `privilege_file: PATH` line to your [YAML config](configuration.md).
+`PATH` represents the path to the privileges file, generally named `privs.json`.
+
+## User and Password Arguments
+
+Before the introduction of users and privileges, Dolt supported only a single user with an accompanying password.
+This was done using the `--user` and `--password` arguments ([see the docs for their defaults](../cli.md#dolt-sql-server), also available using YAML configuration), whereby a server would only allow connections that supplied that singular user and password combination.
+Although Dolt now supports users in a similar fashion to MySQL, we still retain the user and password arguments.
+In MySQL, the default super account (generally called the root user) is created during installation and configuration.
+Rather than creating a parallel with [`init`](../cli.md#dolt-init), we handle the super account creation when starting a server via the arguments.
```

Comment:
(minor) I reread this sentence a couple of times to parse it. I think the "creating a parallel with init" is what threw me. I translated this to:
"Rather than creating this super account during `dolt init`, we instead handle the super account creation when starting a server via the arguments". 

If I understood that correctly, you might consider tweaking that sentence to make it a little clearer.

Reply:
You understood correctly! I'll tweak it